app/models/folio/request.rb

```diff
@@ -99,7 +99,7 @@ def placed_library
 
     def library
       if from_borrow_direct?
-        Settings.BORROW_DIRECT_CODE
```

Comment:
Shouldn't we keep `Settings.BORROW_DIRECT_CODE` to make sure we trigger the right behaviors in the view?

Reply:
I'm not sure. I thought we wanted to stop using the Settings.

Reply:
We're currently using `BORROW_DIRECT` to get the right name mapping here:

https://github.com/sul-dlss/mylibrary/blob/main/config/application.rb#L65

Are we ready to get those mappings from FOLIO now?

Reply:
Ah. Thank you I didn't know this.

Reply:
Would it be appropriate to add "SUL-BORROW-DIRECT" as a mapping and make a note to remove the old one after FOLIO?

Reply:
If it were me, I'd keep using the `Settings` value and post-FOLIO we can just use the FOLIO discovery labels.